src/libraries/System.Private.CoreLib/src/System/Byte.cs

```diff
@@ -606,43 +666,43 @@ static bool INumberBase<byte>.TryConvertFromChecked<TOther>(TOther value, out by
 
             if (typeof(TOther) == typeof(char))
             {
-                char actualValue = (char)(object)value;
+                char actualValue = (char)(object)value!;
```

Comment:
It seems like the type of `value` hasn't been updated. Why is this null-forgiving operator now needed?

Reply:
Its because `where T : Interface` constrains it to be "non-nullable" while `where T : Interface?` would allow it to be null.

I could fix this by adding back the matching constraint and will do so.